src/main/java/eu/xenit/gradle/alfrescosdk/tasks/AmpSourceSetConfiguration.java

```diff
@@ -28,19 +30,9 @@ default AmpSourceSetConfiguration fileMapping(Map<String, String> fileMappingPro
 
     AmpSourceSetConfiguration fileMapping(Action<? super Properties> configure);
 
-    AmpSourceDirectories getConfig();
+    SourceDirectorySet getConfig();
 
-    default AmpSourceSetConfiguration config(Action <? super AmpSourceDirectories> configure) {
```

Comment:
Why have these action methods been removed?
Are they handled automatically by Gradle?

Reply:
First, the execute was commented out. Then I removed the method because it wasn't doing anything anymore.

Reply:
Added back now